airbyte-workers/src/main/java/io/airbyte/workers/process/KubePodProcess.java

```diff
@@ -243,7 +243,7 @@ private static void copyFilesToKubeConfigVolume(final KubernetesClient client,
           // Copying the success indicator file to the init container causes the container to immediately
           // exit, causing the `kubectl cp` command to exit with code 137. This check ensures that an error is
           // not thrown in this case if the init container exits successfully.
-          if (file.getKey().equals(SUCCESS_FILE_NAME) && waitForInitPodToTerminate(client, podDefinition, 10, TimeUnit.SECONDS) == 0) {
+          if (file.getKey().equals(SUCCESS_FILE_NAME) && waitForInitPodToTerminate(client, podDefinition, 5, TimeUnit.MINUTES) == 0) {
```

Comment:
I just picked 5 mins here because that is what we use when waiting for the init pod to run [here](https://github.com/airbytehq/airbyte/blob/73002401543cf6e8ac7a201ca27bcdfd64d2b0da/airbyte-workers/src/main/java/io/airbyte/workers/process/KubePodProcess.java#L276)